Subscriber.php

```diff
@@ -347,6 +347,168 @@ public function maybe_print_update_settings_notice() {
 
 	}
 
+	/**
+	 * Save Cloudflare dev mode admin option.
+	 *
+	 * @since 3.5.2
+	 * @author Soponar Cristina
+	 *
+	 * @param array $devmode New value for Cloudflare dev mode.
+	 */
+	private function save_cloudflare_devmode( $devmode ) {
```

Comment:
If the argument is an array, it should be type-hinted.
```suggestion
	private function save_cloudflare_devmode( array $devmode ) {
```
Same goes for the other methods.

Reply:
Forgot to update the docblock. It's done now

Reply:
#CopyPasteDrama 😏

Reply:
It was from the first refactor and then it ended up in the #CopyPasteDrama 🥇